scripts/postversion.sh

```diff
@@ -5,3 +5,12 @@ git push;
 git push --tags;
 git push git@github.com:paypal/paypal-checkout.git v4;
 npm publish;
+
+# verify new version has been published and then publish to cdnx
```

Comment:
One thought around improving this comment:
```suggestion
# publish to cdnx after verifying a successful publish to the public npm registry
```

Reply:
Updated. Thanks for the suggestion!